### PR TITLE
added new region ca-west-1

### PR DESCRIPTION
**Pull request by atifhafeez**
https://github.com/rancher/rancher/issues/44214

Adding new region ca-west-1 as it is currently not supported.

### CODE REVIEW

**Comment by atifhafeez** [comment]
@jiaqiluo 

Kindly support with this pr. Need to urgently create resources in new aws region.

**Comment by macedogm** [comment]
CC @snasovich and @Sahota1225 for prioritization.

**Comment by atifhafeez** [comment]
> CC @snasovich and @Sahota1225 for prioritization.

@snasovich  @Sahota1225

Kindly advise if it is going to take some time.

**Comment by atifhafeez** [comment]
Kindly advise @macedogm if it is already planned for near future release.

**Comment by atifhafeez** [comment]
@jiaqiluo Kindly have a loot at this, we need this region for deploying cluster.

**Comment by jiaqiluo** [comment]
hi @atifhafeez, thank you for raising the PR. I do not have the permission to merge the PR. Can you submit your request for the new region via the support channel if you have any? That will allow the product management team to prioritize it. Thanks!

**Comment by atifhafeez** [comment]
@jiaqiluo unfortunately, we do not have support channel. Do you know who manages this repo?

**Comment by atifhafeez** [comment]
@macedogm It seems it is still not in priority list. Kindly advise if there is any plan to add ca-west-1 region in this?

**Comment by macedogm** [comment]
@atifhafeez I'm forwarding your comment internally to the appropriate team, as I'm not responsible for this repo.

File: drivers/amazonec2/region.go [hunk 1]
```diff
@@ -22,6 +22,7 @@ var regionDetails map[string]*region = map[string]*region{
 	"ap-south-2":      {"ami-0eaa3ccae04723740"},
 	"ap-east-1":       {"ami-0a7893426ebe74a8c"},
 	"ca-central-1":    {"ami-095509bf36d02a8e0"},
+	"ca-west-1":       {"ami-07cfb3e4b8d29eaf4"},
```
Comment (jiaqiluo) [inline drivers/amazonec2/region.go:25]:
```suggestion
	"ca-west-1":       {"ami-07ce2430f97a1fad1"},
```

The latest AMI for `Ubuntu 20.04 LTS hvm:ebs-ssd (amd64)` is `ami-07ce2430f97a1fad1` on https://cloud-images.ubuntu.com/locator/ec2/. 
Can you update your PR?

**Review by jiaqiluo — CHANGES_REQUESTED** [review]

File: drivers/amazonec2/region.go (hunk 1 above)
Comment (atifhafeez) [reply drivers/amazonec2/region.go]: done

**Review by jiaqiluo — APPROVED** [review]

**Review by snasovich — APPROVED** [review]

**Pull request merged**